contracts/utils/Random.sol

```diff
@@ -22,12 +22,19 @@
 pragma solidity ^0.8.24;
 
 import { IRandom } from "@skalenetwork/professional-interfaces/IRandom.sol";
+import { Precompiled } from "./fieldOperations/Precompiled.sol";
 
 /**
  * @title Random
  * @dev The library for generating of pseudo random numbers
  */
 library Random {
+    /**
+     * @dev Retrieves randomSeed from rng
+     */
+    function randomSeed(address rng) internal view returns (uint256 seed) {
```

Comment:
Don't like you put the function here.
The Random library aims to generate random numbers. Are you sure that the function matches the target?